### DISCUSSION TITLE
Automatically close issue when it is moved to "Done" column in a project

### DISCUSSION THREAD

**Discussion by ssgosh** [community/community#5707] [Projects and Issues]
This issue was already open on the old github repo, but with no info about whether this feature is planned or not.

https://github.com/isaacs/github/issues/1636

Copying description:

"Automation can be set up to move an issue to the "Done" column when the issue is closed. But it would be nice to also have it the other way around: When I'm looking at my Kanban at the end of the day and dragging issues between columns, then issues that I move from "In Progress" to "Done" should be closed automatically. Right now, I have to open up each issue separately to close them."

I will find this feature very useful in managing a project.

**Comment by erikr** [comment]
Yes, this is a much-needed feature!

**Comment by rantoniuk** [comment]
[5 years later and still no progress](https://github.com/dear-github/dear-github/issues/284) for such a basic thing? that's Atlassian style 🙄

**Comment by jagamboa7** [comment]
Can we create an automation for this?

**Comment by ssgosh** [comment]
I just tested this and this seems to work now: https://github.blog/changelog/2024-04-25-github-issues-projects-auto-close-issue-project-workflow/ 

I think for existing projects you may have to manually enable the workflow.

**Reply by Keegnan** [reply]
Can confirm this works and has saved our teams many clicks.
